practice/Reviews/users.txt

```diff
@@ -0,0 +1,4 @@
+{"user_name": "Abby", "hometown": "Portland"}
```

Comment:
Please be careful about what branch you're committing and merging your work on. Running git merge will incorporate that other branch into the currently checked out branch, not always master.